serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,101 @@
+= Migrating Your PostgreSQL Database with a Kogito upgrade
+:flyway_install_url: https://flywaydb.org/download/community
+:flyway_migrate_existing_url: https://flywaydb.org/documentation/learnmore/existing
+:kogito_ddl_script_url: https://repository.jboss.org/org/kie/kogito/kogito-ddl
+:flyway_url: https://flywaydb.org/
+:flyway_baseline_migration_url: https://documentation.red-gate.com/fd/baseline-migrations-184127465.html
+
+Kogito provides an out-of-the-box way to manage your database schema changes with each upgrade. It uses link:{flyway_url}[Flyway] to achieve this functionality.
+
+When you upgrade your Kogito version, by default it will pick up the latest changes available from Kogito and apply them automatically.
+
+NOTE: This feature is currently available only for `kogito-addons-quarkus-persistence-postgresql` extension.
+
+== How to migrate
+
+=== Migrate using Flyway Config
+
+* Kogito provides a default mechanism for migrating your database while updating the Kogito version using the following `Flyway` properties (default value is `false`):
```

Comment:
```suggestion
* Kogito provides a default mechanism for migrating your database while updating the Kogito version using the following `flyway` properties (default value is `false`):
```

Reply:
The technology name should start with capital case as suggested by @MarianMacik [here](https://github.com/kiegroup/kogito-docs/pull/183#discussion_r958082930)

Reply:
then do not use the ` 

```suggestion
* Kogito provides a default mechanism for migrating your database while updating the Kogito version using the following Flyway properties (default value is `false`):
```